modules/ROOT/pages/edit-exchange-json-file.adoc

```diff
@@ -0,0 +1,32 @@
+= Edit your API Specification Project Exchange File
```

Comment:
```suggestion
= Edit Your API Specification Project Exchange File
```
Unless they exceed 4 letters, the words you don't init. cap. are conjunctions, coordinating prepositions, and articles  only.